CATALOG.md

```diff
@@ -179,7 +179,7 @@ Suggested Remediation|Ensure that your Operator is installed via OLM.
 Property|Description
 ---|---
 Version|v1.0.0
-Description|http://test-network-function.com/testcases/operator/install-status Ensures that CNF Operators abide by best practices.  The following is tested: 1. The Operator CSV reports "Installed" status. 2. TODO: Describe operator scc check.
+Description|http://test-network-function.com/testcases/operator/install-status Ensures that CNF Operators abide by best practices.  The following is tested: 1. The Operator CSV reports "Installed" status. 2. Controls the actions that pods created by the operator can perform (e.g. checks that the operator is not configured with privileged rights). If clusterPermissions exists in the CSV, and can perform the clusterPermissions[]. rules[].resourceNames is not empty then we fail the test. Otherwise, the test passes. In case the test fails, we need to analyze the need for the privilege mode.
```

Comment:
the word "can perform" in front of clusterPermissions[] can be removed

Reply:
Not sure if describing the exact logic in English is a good idea. Seems pretty hard to read to me.

Reply:
I see the recommendation from David and Ying. It reads better than the current version which is a strange mix of Shimrit's original with David and Ying's text.